swap_meet/clothing.py

```diff
@@ -0,0 +1,20 @@
+from swap_meet.item import Item
+
+#wave_5
+class Clothing(Item):
+    """
+    inheritance code from method condittion_description 
+
+    """
+    def __init__(self, category = "Clothing", condition = 0):
+        self.category = "Clothing"
+        self.condition = condition 
+
+        super().__init__("Clothing", condition)
```

Comment:
You've done the same work multiple times, consider refactoring like this:
```suggestion
    def __init__(self, condition = 0):
        super().__init__(condition, category="Clothing")
```